tests/store/artifact/test_local_artifact_repo.py

```diff
@@ -243,3 +244,90 @@ def test_trace_data(local_artifact_repo):
     mock_trace_data = {"spans": [], "request": {"test": 1}, "response": {"test": 2}}
     local_artifact_repo.upload_trace_data(json.dumps(mock_trace_data))
     assert local_artifact_repo.download_trace_data() == mock_trace_data
+
+
+@pytest.fixture
+def external_secret_dir(tmp_path):
+    secret_dir = tmp_path.parent / "secrets_outside"
+    secret_dir.mkdir(exist_ok=True)
+    secret_file = secret_dir / "secret.txt"
+    secret_file.write_text("SECRET_CONTENT")
```

Comment:
```suggestion
    secret_file.touch()
```

since contents don't matter